Source/Core/DolphinQt2/DolphinQt2.manifest

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
The msbuild parameter apparently causes this option to be passed to `link`:  
`/manifestinput:"C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\COMMUNITY\VC\TOOLS\MSVC\14.10.25017\INCLUDE\MANIFEST\DPIAWARE.MANIFEST"`
It looks like the toolchain installs a file for each setting:
```
06/01/2017  09:55 PM               363 dpiaware.manifest
06/01/2017  09:55 PM               366 PerMonitorHighDPIAware.manifest
```
So cmake can just do that?